client/src/components/graph/setupCentroidSVG.js

```diff
@@ -0,0 +1,33 @@
+import * as d3 from "d3";
+import styles from "./graph.css";
+
+export default (responsive, graphPaddingRight, xy, text) => {
+  const containerWidth = responsive.width - graphPaddingRight;
+
+  const svg = d3
+    .select("#graphAttachPoint")
+    .append("svg")
+    .attr("id", "centroid")
+    .attr("data-testid", "centroid-overlay")
+    .attr("width", containerWidth)
+    .attr("height", responsive.height)
+    .attr("class", `${styles.graphSVG}`)
+    .style("z-index", 998);
+  //  TODO: Create own styles, ask Colin for an explanation on the css
+  // For now I'm going to put centroid z-index at 998 and lasso on 999
+
+  const label = svg
+    .append("g")
+    .attr("transform", `translate(${xy[0]}, ${xy[1]})`);
+
+  label
+    .append("text")
+    .attr("text-anchor", "middle")
+    .text(text)
+    .style("font-size", "25px")
+    .style("font-family", "Roboto Condensed")
+    .style("fill", "white")
+    .style("stroke", "black");
```

Comment:
```
    .style("font-size", "16px")
    .style("font-weight", "700")
```

Reply:
![image](https://user-images.githubusercontent.com/8716829/59378297-da1b7c80-8d08-11e9-9917-760658affacd.png)

16px is a bit hard to read but I'm going to move forward with it